[2024-07-13T06:42:07.624+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Webscraping.Pre-Scrape_script manual__2024-07-13T06:32:02.128018+00:00 [queued]>
[2024-07-13T06:42:07.640+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Webscraping.Pre-Scrape_script manual__2024-07-13T06:32:02.128018+00:00 [queued]>
[2024-07-13T06:42:07.640+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 2
[2024-07-13T06:42:07.654+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): Pre-Scrape_script> on 2024-07-13 06:32:02.128018+00:00
[2024-07-13T06:42:07.657+0000] {standard_task_runner.py:60} INFO - Started process 172 to run task
[2024-07-13T06:42:07.660+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Webscraping', 'Pre-Scrape_script', 'manual__2024-07-13T06:32:02.128018+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/webscrape_dag.py', '--cfg-path', '/tmp/tmprpv4fun4']
[2024-07-13T06:42:07.663+0000] {standard_task_runner.py:88} INFO - Job 70: Subtask Pre-Scrape_script
[2024-07-13T06:42:07.717+0000] {task_command.py:423} INFO - Running <TaskInstance: Webscraping.Pre-Scrape_script manual__2024-07-13T06:32:02.128018+00:00 [running]> on host 0d386e3ce452
[2024-07-13T06:42:07.800+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Webscraping' AIRFLOW_CTX_TASK_ID='Pre-Scrape_script' AIRFLOW_CTX_EXECUTION_DATE='2024-07-13T06:32:02.128018+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-13T06:32:02.128018+00:00'
[2024-07-13T06:42:07.803+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-13T06:42:07.805+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/ec2-user/pre_scrape.py']
[2024-07-13T06:42:07.819+0000] {subprocess.py:86} INFO - Output:
[2024-07-13T06:42:07.858+0000] {subprocess.py:93} INFO - python3: can't open file '/home/ec2-user/pre_scrape.py': [Errno 2] No such file or directory
[2024-07-13T06:42:07.863+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2024-07-13T06:42:07.876+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2024-07-13T06:42:07.881+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=Webscraping, task_id=Pre-Scrape_script, execution_date=20240713T063202, start_date=20240713T064207, end_date=20240713T064207
[2024-07-13T06:42:07.894+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 70 for task Pre-Scrape_script (Bash command failed. The command returned a non-zero exit code 2.; 172)
[2024-07-13T06:42:07.918+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-07-13T06:42:07.943+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
